src/modules/preview-site/components/preview-site-row.tsx

```diff
@@ -93,20 +94,29 @@ export function PreviewSiteRow( {
 			<div className="flex items-center px-8 py-6">
 				<div className="w-[51%]">
 					<div className="flex items-center">
-						<div className="text-[13px] leading-5 line-clamp-1 break-all">
+						<div
+							className={ cx(
+								'text-[13px] leading-5 line-clamp-1 break-all',
+								isExpired && 'line-through'
```

Comment:
Let's make it `text-a8c-gray-70` when expired.

Reply:
Looking at the design (RToz6tIuQ7nlZrikBte4GU-fi-6541_2932), it is using `text-[#757575]` color when expired:

![Markup on 2025-02-11 at 12:32:35](https://github.com/user-attachments/assets/77aaa478-9f29-4798-a40c-e4047f7c85e1)

→ Made the change in https://github.com/Automattic/studio/pull/913/commits/f4f7d88b9502e6c20ab33bf6b6738b1bbfe03ac3:

![Markup on 2025-02-11 at 12:31:53](https://github.com/user-attachments/assets/2ccc94d0-e262-4ed0-ad8d-d7ff73edf780)


But if you like, we can still make it `text-a8c-gray-70` instead. 🙂